Google.Api.Generator.Tests/ProtoTests/BasicClient.cs

```diff
@@ -119,5 +120,41 @@ public static BasicClient Create(grpccore::CallInvoker callInvoker, BasicSetting
 
         /// <summary>The underlying gRPC Basic client</summary>
         public virtual Basic.BasicClient GrpcClient => throw new sys::NotImplementedException();
+
+        /// <summary>Test summary text for IdempotentMethod</summary>
+        /// <param name="request">The request object containing all of the parameters for the API call.</param>
+        /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
+        /// <returns>The RPC response.</returns>
+        public virtual Response IdempotentMethod(Request request, gaxgrpc::CallSettings callSettings = null) => throw new sys::NotImplementedException();
```

Comment:
How hard would it be to add a line break after `=>`? Definitely not critical, but would reduce the number of places we have huge lines. Definitely one to do later rather than now.

Reply:
Yes, formatting will be done in a later PR, and will use a "proper" line-length-aware formatter that'll make everything look beautiful.
Fortunately that's an orthogonal problem to the code-gen, so can be considered independently.